community/knative/charts/serving/charts/monitoring/templates/monitoring-crb.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
```

Comment:
Need the app, chart, heritage, and release metadata labels here.

Reply:
We will fix this in next commit